packages/dataviews/src/style.scss

```diff
@@ -85,6 +85,10 @@
 	tr {
 		border-bottom: 1px solid $gray-100;
 
+		.dataviews-view-table-header-button {
```

Comment:
We should do a second round of CSS naming consolidation at some point. We do `.dataviews-view-grid__X` but `.dataviews-view-table-X`. Not for this PR, though!